File: ATS.UI/ClientApp/src/app/core/interfaces/registration.ts
export class IRegistration {
    RegistrationID: number;
    FirstName: string;
    LastName: string;
    Email: string;
    Password: string;
    ConfirmPassword: string;
    Phone: string;
    Company: string;
    CompanyURL: string;
    City: string;
    StateID?: number;
    CountryID?: number;
    ZIPCode: string;
    GUID: string;
    IsComplete?: boolean;
    IsActive?: boolean;
    RecordID?: number;
    DateCreated: Date;
    CreatedBy: string;
    DateModified: Date;
    ModifiedBy: string;
   // TimeStamp: Date;
    //ReturnCode: number;
}
export class IRegistrationDisplay {
    registrationID: number;
    firstName: string;
    lastName: string;
    email: string;
    password: string;
    confirmPassword: string;
    phone: string;
    company: string;
    companyURL: string;
    city: string;
    stateID: number;
    countryID: number;
    zipCode: string;
    guid: string;
    isComplete?: boolean;
    isActive?: boolean;
    recordID?: number;
    dateCreated: Date;
    createdBy: string;
    dateModified: Date;
    modifiedBy: string;
    timeStamp: Date;
    //ReturnCode: number;
}

File: ATS.DTO/Registration.cs
using System;

namespace ATS.DTO
{
    public class Registration : AbstractDataObject
    {
        public int RegistrationID 
        {
            get { return base.RecordID; }
            set { base.RecordID = value; }
        }
        public string Email { get; set; }
        public string Password { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string Phone { get; set; }
        public string Company { get; set; }
        public string CompanyURL { get; set; }
        public string City { get; set; }
        public int? StateID { get; set; }
        public string StateName { get; set; }
        public int? CountryID { get; set; }
        public string CountryName { get; set; }
        public string ZIPCode { get; set; }
        public Guid? GUID { get; set; }
        public bool IsActive { get; set; }
        public bool IsComplete { get; set; }
    }
}
